.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,114 @@
+  process-sdk-suppressions-labels:
+    name: Sdk Suppression Labeling
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          # Required since "HEAD^" is passed to Get-ChangedFiles
+          fetch-depth: 2
+
+      - name: Setup Node and run `npm ci`
+        uses: ./.github/actions/setup-node-npm-ci
+
+      - name: Get GitHub PullRequest Changed Files
+        shell: pwsh
+        id: get-changedFiles
+        run: |
+          . eng/scripts/ChangedFiles-Functions.ps1
+          $changedFiles = @(Get-ChangedFiles)
+          echo "PR Changed files: $changedFiles"
+          Add-Content -Path $env:GITHUB_OUTPUT -Value "changedFiles=$changedFiles"
+
+      - name: Get GitHub PullRequest Context
+        uses: actions/github-script@v7
+        id: fetch-pullRequest-context
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            if (!pr) {
+              throw new Error("This workflow must run in the context of a pull request.");
+            }
+            console.log("This action trigger by ", context.eventName);
+            const prContext = {
+              number: pr.number,
+              title: pr.title,
+              html_url: pr.html_url,
+              labels: pr.labels.map(label => label.name)
+            };
+            const prContextStr = JSON.stringify(prContext);
+            console.log("PR Context:", prContextStr);
+            core.setOutput("prContext", prContextStr);
+          result-encoding: string
+
+      - name: Run Get suppressions label script
+        id: run-suppressions-script
+        env:
+          GITHUB_PULL_REQUEST_CONTEXT: ${{ steps.fetch-pullRequest-context.outputs.prContext }}
+          OUTPUT_FILE: "output.json"
+          GITHUB_PULL_REQUEST_CHANGE_FILES: ${{ steps.get-changedFiles.outputs.changedFiles }}
+        run: |
+          node eng/tools/sdk-suppressions/cmd/sdk-suppressions-label.js
+          OUTPUT=$(cat $OUTPUT_FILE)
+          echo "Script output labels: $OUTPUT"
+
+          labelsToAdd=$(echo "$OUTPUT" | sed -n 's/.*"labelsToAdd":\[\([^]]*\)\].*/\1/p' | tr -d '" ')
+          labelsToRemove=$(echo "$OUTPUT" | sed -n 's/.*"labelsToRemove":\[\([^]]*\)\].*/\1/p' | tr -d '" ')
+
+          for label in $(echo $labelsToAdd | tr ',' '\n'); do
+            echo "Label to add: $label"
+            echo "has-$label=true" >> $GITHUB_OUTPUT
+            echo "$label=true" >> $GITHUB_OUTPUT
+          done
+
+          for label in $(echo $labelsToRemove | tr ',' '\n'); do
+            echo "Label to remove: $label"
+            echo "has-$label=true" >> $GITHUB_OUTPUT
+            echo "$label=false" >> $GITHUB_OUTPUT
+          done
+
+      - uses: ./.github/actions/add-label-artifact
+        name: Upload artifact with results-go
+        if: ${{ steps.run-suppressions-script.outputs.has-BreakingChange-Go-Sdk-Suppression == 'true' }}
```

Comment:
```suggestion
```

You probably want to always set the value to either `true` or `false` rather than "not set", to handle the case where a label needs to be removed, if it no longer applies after a later check run.

Reply:
correct ! Changed